includes/reader-revenue/woocommerce/class-woocommerce-cover-fees.php

```diff
@@ -76,11 +76,7 @@ public static function add_transaction_fee( $cart ) {
 			return;
 		}
 		$cart->add_fee(
-			sprintf(
```

Comment:
I'm actually not sure in what other contexts we might be showing this fee name within Newspack, so it's possible we may need to go with another solution if the `($12.34)` bit of the fee name is needed elsewhere. I wasn't able to find anywhere else this was used, but let me know if I've missed something.